admin/src/collections/Messages.ts

```diff
@@ -0,0 +1,39 @@
+import { buildProperties } from '@camberi/firecms';
+import { Message, MESSAGE_FIRESTORE_PATH } from '../../../shared/src/types';
+import { buildAuditedCollection } from './shared';
+
+export const messagesCollection = buildAuditedCollection<Message>({
+	name: 'Messages',
+	group: 'Messages',
+	path: MESSAGE_FIRESTORE_PATH,
+	icon: 'SupervisorAccountTwoTone',
+	description: 'Lists all messages for one recipient or user',
+	customId: true,
+	properties: buildProperties<Message>({
+		type: {
+			dataType: 'string',
+			name: 'Type',
+			enumValues: {
+				sms: 'SMS',
+				email: 'EMAIL',
+			},
+		},
+		sent_at: {
+			dataType: 'date',
+			name: 'Message Date',
+			mode: 'date',
+		},
+		content: {
```

Comment:
Here, we then could add later a custom view which renders html, so that we can display the html emails.